deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,58 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+
+body {
+    border: 3px dashed red;
+
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+/*////////////////////Navigatie////////////////////////////*/
+
+.col-1 {
+    flex-basis: 250px;
+    flex-grow: 0;
+    flex-shrink: 0;
+}
+
+/*//////////////////Basis/////////////////////////////*/
+
+.col-2 {
+    flex-basis: 70vw;
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    flex-shrink: 0;
+}
+
+.col-2 header {
+    flex-basis: 150px;
+    flex-grow: 0;
+    flex-shrink: 0;
+}
+
+.col-2 main {
+    display: flex;
+    flex-basis: auto;
+    flex-grow: 1;
+    flex-shrink: 1;
+
+}
+
+.col-2 article {
+    flex-basis: auto;
```

Comment:
hier had je beter 20vw kunnen gebruiker. het element mag niet groter worden dan de basis afmeting als er meer ruimte is.